src/omnicore/doc/release-notes.md

```diff
@@ -1 +1,528 @@
+```
+{
+  "address": "3Fc5gWzEQh1YGeqVXH6E4GDEGgbZJREJQ3",
+  "redeemScript": "542102d797b8526701a3dfdb52d11f89377e0288b14e29b1414d64de065cd337069c3b21036a4caa95ec1d55f1b75a8b6c7345f22b4efc9e25d38ab058ef7d6f60b3b744f7410499e86235a6a98fc295d7cfe641d37409f2840ad32e0211579cae488bd86cf01daf7ad8f082d968ea4bca77e794fffd1a31583f36f37b1198e51d0651cdbcf3214104b7a3d7f7ccdf211dfd180815b87332b4773cc40bff72a4d0bb60f3a85409d19f99709331c6b11c976fe274a86d789a1cf2b3b0be29fe5fc55c93ad9e08459c4f4104e65b098558d637cfcf3194214637f8838338b141259b698d2a027b069d405b6502ad4a4e9aa75094fa431a6c9af580f5917834a6d4cec946054df33194b2967855ae"
+}
+```
+
+The activation message must be signed by at least 4 of the 5 nominated members of the Omni team in order to be accepted. These members are as follows:
+
+- Zathras - Project Maintainer and Developer
+- dexx - Project Maintainer and Developer
+- J.R. Willett - Founder and Board Member
+- Craig Sellars - Technologist and Board Member
+- Adam Chamely - Project Maintainer and Developer
+
+The transaction format of activation messages is as follows:
+
+| **Field**                  | **Type**        | **Example**                        |
+| -------------------------- | --------------- | ---------------------------------- |
+| Transaction version        | 16-bit unsigned | 65535                              |
+| Transaction type           | 16-bit unsigned | 65534                              |
+| Feature identifier         | 16-bit unsigned | 1 (Class C activation)             |
+| Activation block           | 32-bit unsigned | 385000                             |
+| Min. client version        | 32-bit unsigned | 1000000 (Omni Core 0.0.10)         |
+
+Should a particular client version not support a new feature when it is activated, the client will shut down to prevent providing potentially inaccurate data. In this case the client should be upgraded. Whilst this behavior can be overridden with the configuration option `-overrideforcedshutdown`, it is strongly recommended against.
+
+Senders of activation messages can be whitelisted or blacklisted with the configuration options `-omnialertallowsender=<sender>` and `-omnialertignoresender=<sender>`. Please be aware that overriding activations may cause the client to provide inaccurate data that does not reflect the rest of the network.
+
+The GUI displays warnings for pending activations and daemon users can view pending and completed activations via the [omni_getactivations](https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md#omni_getactivations) JSON-RPC API call.
+
+Consensus affecting changes
+===========================
+
+Starting with this version, all changes of the consensus rules are enabled by activation transactions.
+
+Please note, while Omni Core 0.0.10 contains support for the new rules and features, they are not enabled immediately after the release, but going to be activated via the feature activation mechanism described above.
```

Comment:
Minor (very minor) nit, at this point in the document we haven't said what "the new rules and features" are so a generalization may be appropriate:

> Please note, while Omni Core 0.0.10 contains support for the new rules and features, they are not enabled immediately after the release, but going to be activated via the feature activation mechanism described above.

to

> Please note, while Omni Core 0.0.10 contains support for several new rules and features they are not enabled immediately and will be activated via the feature activation mechanism described above.